### ISSUE TITLE
A11Y | There is no visual indication of keyboard focus when using TAB or SHIFT+TAB to navigate through the submenu dropdown element.

### ISSUE THREAD

**Issue by hilcurtis** [Esri/global-nav#338]
**ID**
1847345824

**Violation**
Ensure keyboard focus is indicated visually

**Module Name**
03b Header Menu - Bottom Row

**Description**
[Issue]
There is no visual indication of keyboard focus when using TAB or SHIFT+TAB to navigate through the submenu dropdown element. 

[User Impact]
Keyboard-only users can become disoriented when focus is not indicated.

[Code Reference]
```
<button class="js-subnav-expand subnav-expand ">
        <span class="js-subnav-current-label subnav-expand__current-label no-pointer inline-block margin-right-half margin-left-0">Overview</span>
        <svg class="subnav-expand-icon no-pointer" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
          <path fill="#FFF" d="M8 11.5l-7-7h14z"></path>
        </svg>
      </button>
```

```
<style>
@media screen and (max-width: 479px)
.subnav-expand.is-active, .subnav-expand:hover, .subnav-expand:focus {
    outline: none;
}
</style>

```
**Note**
 [Recommendation]
Ensure that when actionable elements receive focus, that it appears on screen and that a visible focus indicator is present. Focus can be indicated different ways including change of background, lightness, border, outline, underline, and other visual methods. Avoid the use of the `"outline: none"` CSS property. As of WCAG 2.1, a custom focus indicator should have a minimum contrast ratio of 3:1 against the background.

[Compliant Code Example]
Example of visible keyboard focus outline:
```
<style type="text/CSS">
.subnav-expand.is-active, .subnav-expand:hover, .subnav-expand:focus {
    background-color: #00000040;
    border-bottom-color: rgba(0, 0, 0, 0.4);
}
</style>
```

![image](https://user-images.githubusercontent.com/34460455/100177240-882bcf80-2e86-11eb-8715-9d34847ac4bb.png)
